### ISSUE TITLE
Release with .NET 4.5.2?

### ISSUE THREAD

**Issue by ariccio** [chromium/vs-chromium#15]
I have .NET 4.5.2 installed, yet vs-chromium won't install because I do not have 4.5 installed. The solution configuration targets .NET 4.5, not 4.5.2 - any reason why?

(sorry! I'm not a .NET developer, so I don't know anything about .NET releases)

**Comment by rpaquay** [comment]
As far as I know, a minor release of .net (such as 4.5.2) should not break anything. Can you describe in more details the problem you are running into?

**Comment by ariccio** [comment]
Here's what I got when installing it:
![vschromiumfailed](https://cloud.githubusercontent.com/assets/2142308/8200365/13cf87ee-1492-11e5-91a5-39e6f2247681.PNG)
then this dialog:
![vschromiumdotnet](https://cloud.githubusercontent.com/assets/2142308/8200367/181be50e-1492-11e5-9740-8f472719a8af.PNG)


The contents of the log file:

```
6/17/2015 1:40:41 AM - Microsoft VSIX Installer
6/17/2015 1:40:41 AM - -------------------------------------------
6/17/2015 1:40:42 AM - Initializing Install...
6/17/2015 1:40:42 AM - Extension Details...
6/17/2015 1:40:42 AM - 	Identifier      : a10cf7af-5f0a-4502-b44b-51ff1b7c8a87
6/17/2015 1:40:42 AM - 	Name            : VsChromium
6/17/2015 1:40:42 AM - 	Author          : The Chromium Authors
6/17/2015 1:40:42 AM - 	Version         : 0.9.14
6/17/2015 1:40:42 AM - 	Description     : Collection of tools to help contributing code to the Chromium project.
6/17/2015 1:40:42 AM - 	Locale          : en-US
6/17/2015 1:40:42 AM - 	MoreInfoURL     : 
6/17/2015 1:40:42 AM - 	InstalledByMSI  : False
6/17/2015 1:40:42 AM - 	SupportedFrameworkVersionRange : [4.5]
6/17/2015 1:40:42 AM - 
6/17/2015 1:40:42 AM - 	Supported Products : 
6/17/2015 1:40:42 AM - 		Microsoft.VisualStudio.Pro
6/17/2015 1:40:42 AM - 			Version : [12.0]
6/17/2015 1:40:42 AM - 
6/17/2015 1:40:42 AM - 	References      : 
6/17/2015 1:40:42 AM - 		-------------------------------------------------------
6/17/2015 1:40:42 AM - 		Identifier   : Microsoft.VisualStudio.MPF.12.0
6/17/2015 1:40:42 AM - 		Name         : Visual Studio MPF 12.0
6/17/2015 1:40:42 AM - 		Version      : [12.0]
6/17/2015 1:40:42 AM - 		MoreInfoURL  : 
6/17/2015 1:40:42 AM - 		Nested       : No
6/17/2015 1:40:42 AM - 
6/17/2015 1:40:42 AM - 
6/17/2015 1:40:42 AM - Searching for applicable products...
6/17/2015 1:40:42 AM - Found installed product - Microsoft Visual Studio Professional 2013
6/17/2015 1:40:45 AM - The extension with ID 'a10cf7af-5f0a-4502-b44b-51ff1b7c8a87' is not installed to Microsoft Visual Studio Professional 2013.
6/17/2015 1:40:45 AM - Found installed product - Microsoft Visual Studio 2013 Shell (Integrated)
6/17/2015 1:40:45 AM - Found installed product - Global Location
6/17/2015 1:40:47 AM - The following target products have been selected...
6/17/2015 1:40:47 AM - 	Microsoft Visual Studio Professional 2013
6/17/2015 1:40:47 AM - 
6/17/2015 1:40:48 AM - Beginning to install extension to Microsoft Visual Studio Professional 2013...
6/17/2015 1:40:48 AM - Install Error : Microsoft.VisualStudio.ExtensionManager.MissingTargetFrameworkException: The extension 'VsChromium' requires a version of the .NET Framework that is not installed.
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.CheckForValidDotNetFramework(IExtension extension)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.CheckForInstallBlockers(InstallableExtensionImpl extension, IInstalledExtensionList modifiedInstalledExtensionsList, Boolean isNestedExtension, Boolean& olderVersionInstalled)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.InstallInternal(InstallableExtensionImpl extension, Boolean perMachine, Boolean isNestedExtension, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, AsyncOperation asyncOp, UInt64 totalBytesToWrite, UInt64& totalBytesWritten)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.BeginInstall(IInstallableExtension installableExtension, Boolean perMachine, AsyncOperation asyncOp)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionManagerService.InstallWorker(IInstallableExtension extension, Boolean perMachine, AsyncOperation asyncOp)

```

**Comment by rpaquay** [comment]
Thanks. This should be fixed with version 0.9.15. Please re-open if the problem persists.

**Comment by ariccio** [comment]
I don't think I mentioned it: 0.9.15 fixed the problem! Thanks!
